apps/asynchromix/powermixing.py

```diff
@@ -166,9 +165,14 @@
 if __name__ == "__main__":
     from honeybadgermpc.config import HbmpcConfig
 
+    HbmpcConfig.load_config()
```

Comment:
Not exactly sure how to fix this code-- I couldn't figure out how to get it running.